src/utilities/emailSender.js

```diff
@@ -1,71 +1,59 @@
 const nodemailer = require('nodemailer');
+const { google } = require('googleapis');
 const logger = require('../startup/logger');
 
```

Comment:
We can't have a refresh token that expires in 7 days, normally they should only expire if:
1. you change the email's password
2. you don't use the token for 6 months

According to this:
https://developers.google.com/identity/protocols/oauth2

It says this: A Google Cloud Platform project with an OAuth consent screen configured for an external user type and a publishing status of "Testing" is issued a refresh token expiring in 7 days.

So it looks like we need to change the status from Testing before we deploy so the refresh token we get doesn't expire.

Can you undo the closure changes and then I will test? i.e. integrate your new code into that closure as gmail blocks you if you send emails too fast which will happen at the end of a week when we send out blue square emails.  Let me know if you need help doing this.

Reply:
> We can't have a refresh token that expires in 7 days, normally they should only expire if:
> 
> 1. you change the email's password
> 2. you don't use the token for 6 months
> 
> According to this: https://developers.google.com/identity/protocols/oauth2
> 
> It says this: A Google Cloud Platform project with an OAuth consent screen configured for an external user type and a publishing status of "Testing" is issued a refresh token expiring in 7 days.
> 
> So it looks like we need to change the status from Testing before we deploy so the refresh token we get doesn't expire.
> 
> Can you undo the closure changes and then I will test? i.e. integrate your new code into that closure as gmail blocks you if you send emails too fast which will happen at the end of a week when we send out blue square emails. Let me know if you need help doing this.

I think I have already change the status of our GCP to prod stage and it will not expire in 7 days. Sure will update it using intervals.